docfx_project/ziti/overview.md

```diff
@@ -5,9 +5,60 @@ uid: zitiOverview
 # Overview
 
 ## Why Ziti?
-Ziti represents the next generation of secure networking for your applications and has arrived with the Ziti platform,
-created by NetFoundry, Inc ! The Ziti Controller and the Ziti Edge Router combine to form a secure, Zero Trust
-entry point into your network or as a secure zero-trust proxy to other hosted services.
+Ziti represents the next generation of secure, open-source networking for your applications and has arrived with the Ziti platform, created by NetFoundry, Inc ! The Ziti Controller and the Ziti Edge Router combine to form a secure, Zero Trust entry point into your network or as a secure zero-trust proxy to other hosted services.
+
+Ziti provides several security features, including:
+
+* Zero Trust and Service Segmentation
+* Dark Services and Routers
+* End to end encryption
+
+It also provides performance and reliability through:
+
+* A scalable, extensible mesh fabric with smart routing
+* Support for load balancing services for both horizontal scale and failover setups 
+
+Finally, it offers convenience:
```

Comment:
rather than `convenience` we should highlight that it is developer oriented with REST API, etc